### PR TITLE
News118: add correction to unknown message item

### PR DESCRIPTION
**Pull request by harding**
Contains the commit from #495 by @MarcoFalke

### CODE REVIEW

File: _posts/en/newsletters/2020-10-07-newsletter.md [hunk 1]
```diff
@@ -108,6 +108,14 @@ release candidates.*
     for previous discussion about this proposed method for protocol
     feature negotiation.
 
+    *Correction:* this item originally incorrectly stated that Bitcoin
+    Core penalized peers who sent unknown messages after their `verack`
+    messages.  We've corrected it to indicate that unknown messages were
+    never specifically penalized, although messages received out of the
+    expected order were penalized.  Before this change, that included
+    receiving any message besides `version` before `verack`  We thank
```
Comment (bitschmidty) [inline _posts/en/newsletters/2020-10-07-newsletter.md:116]: s/before `verack`  We/before `verack`.  We

**Review by bitschmidty — COMMENTED** [review]
- reviewed content changes
- ensured successful build

File: _posts/en/newsletters/2020-10-07-newsletter.md [hunk 2]
```diff
@@ -108,6 +108,14 @@ release candidates.*
     for previous discussion about this proposed method for protocol
     feature negotiation.
 
+    *Correction:* this item originally incorrectly stated that Bitcoin
+    Core penalized peers who sent unknown messages after their `verack`
+    messages.  We've corrected it to indicate that unknown messages were
+    never specifically penalized, although messages received out of the
+    expected order were penalized.  Before this change, that included
+    receiving any message besides `version` before `verack`.  We thank
```
Comment (jnewbery) [inline _posts/en/newsletters/2020-10-07-newsletter.md:116]: `wtxidrelay` was also allowed before `verack` when this was merged.

**Review by jnewbery — COMMENTED** [review]
One minor correction, but otherwise this looks good.

Sorry for missing this in review :(

**Comment by jnewbery** [comment]
Looks good to me. Thanks @MarcoFalke and @harding.

**Pull request merged**